app/views/metadata_presenter/analytics/_ga4.html.erb

```diff
@@ -1,16 +1,8 @@
-<!-- Google Analytics 4 -->
+<!-- Form Owner Google Analytics 4 -->
 <% if Rails.application.config.respond_to?(:global_ga4) %>
-<script>
-  gtag('config', '<%= measurement_id %>');
-</script>
-<% else %>
-<script async src="https://www.googletagmanager.com/gtag/js?id=<%= measurement_id %>"></script>
-<script>
-  window.dataLayer = window.dataLayer || [];
-  function gtag(){dataLayer.push(arguments);}
-  gtag('js', new Date());
-
-  gtag('config', '<%= measurement_id %>');
-</script>
+  <%# Global Google Analytics (GA4) should be included before this point so we only need a config setting %>
+  <script>
+    gtag('config', '<%= measurement_id %>');
+  </script>
```

Comment:
this would mean that the global GA4 _has_ to exist? in other words, it would not be possible for us to remove the global analytics but to allow a form owner to use GA4.